lib/manageiq/messaging/kafka/common.rb

```diff
@@ -23,7 +23,7 @@ def consumer(beginning, options)
 
         def raw_publish(body, options)
           options[:payload] = encode_body(options[:headers], body)
-          producer.produce(**options)
+          producer.produce(options)
```

Comment:
Is this a bug that was fixed?

Reply:
Oh yes, missed this when resolving a conflict